frontend/src/components/sightings/sightings.tsx

```diff
@@ -1,26 +1,45 @@
 import { useEffect, useState } from "react";
 import {
-  fetchSightings,
-  SightingResponse,
+  fetchSightings as fetchInternalSightings,
+  SightingResponse as internalSightingResponse,
```

Comment:
This is a type, so should begin with a capital letter
```suggestion
  SightingResponse as InternalSightingResponse,
```